### ISSUE TITLE
Database feedback

### ISSUE THREAD

**Issue by marlycormar** [pharmaR/risk_assessment#122]
The `db_ins` and `db_fun` should provide feedback into how the operation went, i.e., what is successful or not? 

This is not an issue now since we are still developing the app, but in the future when multiple users interact with the app, it may happen that writing/reading to the db fails.

**Comment by Robert-Krajcik** [comment]
Hi @marlycormar 

There are some things we can do to improve the message handling.
for example, if you had a query like
`query <- "select * from mypackage where score < 0.1"`
The message you would see is:
`Error: no such table: mypackage`

But there is no context to it.
Using
```{r, eval=FALSE}
  tryCatch({
  rs <- dbSendQuery(con, query)
  }, error = function(err) {
    message(paste("db_fun query returned",err))
    dbDisconnect(con)
  })
```
We can give it some context, log the message, and close the connection.
We could also issue a message (and log it) if no rows were returned from the query.

**Comment by AARON-CLARK** [comment]
closed with #131
